SoftLayer/managers/storage_utils.py

```diff
@@ -903,6 +906,9 @@ def prepare_duplicate_order_object(manager, origin_volume, iops, tier,
     if volume_is_performance:
         duplicate_order['iops'] = iops
 
+    if dependent_duplicate:
+        duplicate_order['isDependentDuplicateFlag'] = 1
```

Comment:
Would be simpler to just have order['isDependentDuplicateFlag'] = dependent_duplicate and remove the if statement

Reply:
as we discussed, this isn't possible